### DISCUSSION TITLE
v0.41.0

### DISCUSSION THREAD

**Discussion by wata727** [terraform-linters/tflint#1524] [Announcements]
### Enhancements

- [#1519](https://github.com/terraform-linters/tflint/pull/1519): terraform: Add support for Terraform v1.3 ([@wata727](https://github.com/wata727))

### Chores

- [#1510](https://github.com/terraform-linters/tflint/pull/1510): Move evaluation impl to under the terraform package ([@wata727](https://github.com/wata727))
- [#1523](https://github.com/terraform-linters/tflint/pull/1523): Update Terraform compatibility notices ([@wata727](https://github.com/wata727))

<hr /><em>This discussion was created from the release <a href='https://github.com/terraform-linters/tflint/releases/tag/v0.41.0'>v0.41.0</a>.</em>

**Comment by gauravsingh089** [comment]
Hi ,
tflint v0.41.0 is not available in chocolatey for installation in Windows. 
Please advise how to install it in Windows. Thanks

**Reply by wata727** [reply]
The chocolatey package is not maintained by us. Please contact the package maintainer.
Alternatively, you can download it from GitHub releases.
https://github.com/terraform-linters/tflint/releases/tag/v0.41.0
